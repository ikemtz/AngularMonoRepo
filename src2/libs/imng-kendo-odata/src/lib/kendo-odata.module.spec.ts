import { TestBed } from '@angular/core/testing';
import { ImngKendoODataModule } from './kendo-odata.module';

describe('ImngKendoODataModule', () => {
  beforeEach(() => {
    TestBed.configureTestingModule({
      imports: [ImngKendoODataModule],
    }).compileComponents();
  });

  it('should create', () => {
    expect(ImngKendoODataModule).toBeDefined();
  });


  it('should initialize', () => {
    const result = ImngKendoODataModule.forRoot();
    expect(result).toMatchSnapshot();
  });
});
