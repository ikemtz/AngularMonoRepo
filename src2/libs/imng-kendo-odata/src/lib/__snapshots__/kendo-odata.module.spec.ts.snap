// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ImngKendoODataModule should initialize 1`] = `
Object {
  "ngModule": [Function],
  "providers": Array [
    [Function],
  ],
}
`;
